delayScanPlots.py

```diff
@@ -1,48 +1,81 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
-
+import sys
 import matplotlib.colors as mcolors
+import pandas as pd
+import mplhep as hep
+plt.style.use(hep.style.CMS)
+plt.rcParams.update({'font.size': 18})
```

Comment:
Is this increasing or reducing the font size?  In your newer plots, the font is smaller, which we don't want (it'll be harder to read in slides)